app/services/hyrax/multiple_membership_checker.rb

```diff
@@ -53,11 +53,15 @@ def check(collection_ids:, include_current_members: false)
     def single_membership_collections(collection_ids)
       return [] if collection_ids.blank?
 
-      ::Collection.where(:id => collection_ids, Hyrax.config.collection_type_index_field.to_sym => collection_type_gids_that_disallow_multiple_membership)
+      field_pairs = {
+        :id => collection_ids,
+        Hyrax.config.collection_type_index_field.to_sym => collection_type_gids_that_disallow_multiple_membership
+      }
+      Hyrax::FindObjectsViaSolrService.find_for_model_by_field_pairs(model: ::Collection, field_pairs: field_pairs, use_valkyrie: true)
     end
 
     def collection_type_gids_that_disallow_multiple_membership
-      Hyrax::CollectionType.gids_that_do_not_allow_multiple_membership
+      Hyrax::CollectionType.gids_that_do_not_allow_multiple_membership.map(&:to_s)
```

Comment:
what's going on here?

Reply:
Tests initially failed when refactoring for the new approach which required this change.  But the final refactor really doesn't need this.  So I set it back to what it was.